core/primitives-core/src/version.rs

```diff
@@ -33,6 +33,28 @@ pub enum ProtocolFeature {
     /// Restore receipts that were previously stuck because of
     /// <https://github.com/near/nearcore/pull/4228>.
     RestoreReceiptsAfterFixApplyChunks,
+    /// Minimum protocol version for NEP-92
+    MinProtocolVersionNep92,
+    /// Minimum protocol version for NEP-92 fix
+    MinProtocolVersionNep92Fix,
+    /// Creates a unique random seed to be provided to `VMContext` from a given `action_hash` and a given `random_seed`
+    CorrectRandomValue,
+    /// The protocol version that enables reward on mainnet
+    EnableInflation,
+    /// Fix upgrade to use the latest voted protocol version instead of the current epoch protocol
+    /// version when there is no new change in protocol version
+    UpgradabilityFix,
+    /// Updates the way receipt ID, data ID and random seeds are constructed
+    CreateHash,
+    /// Fix the storage usage of the delete key action
+    DeleteKeyStorageUsage,
+    /// Upgrade for shard chunk header
+    ShardChunkHeaderUpgrade,
+    /// Updates the way receipt ID is constructed to use current block hash instead of last block hash
+    CreateReceiptIdSwitchToCurrentBlock,
+    /// Pessimistic gas price estimation uses a fixed value of `minimum_new_receipt_gas` to stop being
+    /// tied to the function call base cost
+    FixedMinimumNewReceiptGas,
```

Comment:
Nit: sort these in approximate ordering of the versions from `protocol_version`.